config/urls.py

```
@@ -1,25 +1,34 @@
+handler500 = handler_server_error_500
 urlpatterns = [
     re_path(r"^$", Index.as_view(), name="index"),
     path("admin/", admin.site.urls),
-    path("restaurant/", include("resturant.urls")),
+    path("restaurant/", include("applications.resturant.urls")),
     path("", include("django_prometheus.urls")),
     path("auth/", include("djoser.urls")),
     path("auth/", include("djoser.urls.authtoken")),
     re_path(r"^checkup/", include("health_check.urls")),
+    path("byte_patrol/", include("applications.byte_patrol.urls")),
 ] + debug_toolbar_urls()
```

Ruff (0.11.9)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)